src/components/podcastAlbum/PodcastAlbumCollectionTabs.tsx

```diff
@@ -0,0 +1,58 @@
+import { Skeleton, Tabs } from 'antd'
+import { useIntl } from 'react-intl'
+import { useApp } from '../../contexts/AppContext'
+import hasura from '../../hasura'
+import { commonMessages } from '../../helpers/translation'
+import { usePodcastAlbumCounts } from '../../hooks/podcastAlbum'
+import AdminCard from '../admin/AdminCard'
+import PodcastAlbumCollectionTable from './PodcastAlbumCollectionTable'
+
+const PodcastAlbumCollectionTabs: React.VFC = () => {
+  const { formatMessage } = useIntl()
+  const { id: appId } = useApp()
+
+  const { loadingPodcastAlbumCounts, counts } = usePodcastAlbumCounts(appId)
+
+  const tabContents: {
+    key: string
+    tab: string
+    condition: hasura.GET_PODCAST_ALBUM_PREVIEW_COLLECTIONVariables['condition']
+  }[] = [
+    {
+      key: 'published',
+      tab: formatMessage(commonMessages.status.published),
+      condition: {
+        published_at: { _is_null: false },
+        app_id: { _eq: appId },
+        is_deleted: { _eq: false },
```

Comment:
we've already set permission on `podcast_album`, not need to add these condition

Reply:
X-Hasura-User-Id: memberId
X-Hasura-App-Id: appId

```graphql
program_content(where: {
    program_content_section: {
        program: {
            app_id: { _eq: $appId }
            author_id: {_eq: $currentMemberId}
        }
    }
}) {
    id
    title
}
```

```sql
select program_content.id, program_content.title
from program_content
join program_content_section on ....
join program on ....
where program.app_id = $appId
```

```sql
with permitted_program_content as (
    select *
    from program_content
    where [permission]
) 
select id, title
from permitted_program_content
```